src/app/services/article.service.ts

```diff
@@ -73,6 +79,51 @@ export class ArticleService {
     return this.articles;
   };
 
+  public fetchAllArticlesFromFirestore(): Observable<Article[]> {
```

Comment:
We should not fetch articles we're not going to display. If we're going this, it's probably better to just commit the json to git. Maybe have a job that commits (deploy) it on a regular basis.

Reply:
This having a github workflow to do this might actually solve the issue that Chris brought up... so preview will display what's in memory or what's in firestore... and will not be deployed until it's committed to git.

The documents will also be versioned (git versioned).